ocaml/xapi/config_file_sync.ml

```diff
@@ -48,7 +53,9 @@ let transmit_config_files s =
    the entire /etc/password file. We need to make sure we send the
    "un-shadowed" passwd file, so that slaves don't overwrite root's
    password with an 'x'. *)
-let legacy_transmit_passwd s = unshadow () |> write_to_fd s
+let legacy_transmit_passwd s =
```

Comment:
Is this comment still true? If so, it'd be helpful to know here when it will become false

Reply:
We can't rely on tag numbers before Ely to make a lot of sense unfortunately. A big of digging in the history suggests that, while this commit went into master quite long ago, just after the release of Clearwater, the first release it got into was Dundee. This is probably because Clearwater-Creedence-Cream were separate from master for quite long.

Reply:
In other words, we will still need this legacy function for a while longer to support upgrades.

Reply:
OK, while we're in the area can we adjust the comment to state precisely when we can remove the code?